utils/views.py

```diff
@@ -0,0 +1,129 @@
+import os
+import logging
+from configobj import ConfigObj
+
+from django.views.generic.edit import FormView
+from django.views.generic import TemplateView
+from django.core.validators import validate_email, ValidationError
+from django.shortcuts import redirect
+
+from wam.settings import BASE_DIR, WAM_EXCHANGE_ACCOUNT, WAM_EXCHANGE_EMAIL, WAM_EXCHANGE_PW
+from utils.forms import FeedbackForm
+from utils.mail import send_email
+
+
+class FeedbackView(FormView):  # pylint: disable=too-many-ancestors
```

Comment:
If we decide to ignore such warning, we can also write that in the [.pylintrc file](https://github.com/rl-institut/WAM/blob/5484632186cbbab83468778ccb8791e7fbd2d459/.pylintrc#L57)

Reply:
In this case it's a django issue but I'd generally keep it in the `.pylintrc` as it may points us to app-related problems in the future..